### ISSUE TITLE
Device offline half of the time, nothing changed

### ISSUE THREAD

**Issue by bass63** [berezhinskiy/ecoflow_exporter#64]
After 11 of June device half of the time the device is offline and the graphs are full of gaps

![IMG_0160](https://github.com/user-attachments/assets/027d5272-2fe9-429d-a34e-2953d0581605)

Was working fine for a month before. Restarting the docker fixed everything for one day and then the problem came back and now it’s like that all of the time, reboot doesn’t help. Here are the logs

[ecoflow_exporter.csv](https://github.com/user-attachments/files/16230771/ecoflow_exporter.csv)

[prometheus.csv](https://github.com/user-attachments/files/16230803/prometheus.csv)

Can someone help? Is this a EcoFlow server problem? EcoFlow official app works fine.

**Comment by pontostroy** [comment]
Try code from this MR
https://github.com/berezhinskiy/ecoflow_exporter/pull/57/files
It has updated mqtt libs

**Comment by bass63** [comment]
Thanks! But I’m not a coder, merely an experienced user, so I can’t compile. Is there a new docker version or some archive with files I could replace inside a container?

**Comment by scamianbas** [comment]
Hi, the gaps are wifi/internet connection drops IMHO

**Comment by bass63** [comment]
> Hi, the gaps are wifi/internet connection drops IMHO

Nope, the unit is right next to router, the internet is fine, I’m streaming video from Plex server at the same time

**Comment by online** [comment]
I have the same issue 😕

**Comment by johsk** [comment]
I also have the same issue.

**Comment by KulykDmytro** [comment]
i beleive same here: #67

**Comment by bass63** [comment]
> Well, the problem has gone for me for last 20 hours, without me doing anything to this setup. I did install a homeassistant docker and it also reads EcoFlow Cloud info and maybe it keeps a connection alive? Or maybe it was server problems all the time. Anyway, I’ll keep updating this thread for a few days to see if the problem went away completely

Nope, that lasted less than a day. Now we know from numerous other posts that it is a widespread problem with API access, hope someone will fix it
